### PR TITLE
MARS Dataset

### PR DESCRIPTION
**Pull request by bar371**
Create MARS-like dataset from our data

### CODE REVIEW

File: utils/build_video_reid_dataset.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+def im_name_in_mars(crop: Crop, track_counter, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}C{crop.cam_id}T{track_counter:04d}F{crop_id:03d}.png'
+
+def im_name_in_duke(crop: Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_C{crop.cam_id}_F{crop_id + 1:04d}_X{crop.frame_num:05d}.png'
+
+def im_name_in_img_duke(crop:Crop, crop_id):
+    return f'{NAME_TO_ID[crop.label]:04d}_c{crop.cam_id}_f{crop_id:07d}.jpg'
+
+def convert_to_img_reid_duke_naming(dataset_path:str, query_days, same_day=False):
```
Comment (darkushin) [inline utils/build_video_reid_dataset.py]: we have the file `utils/build_reid_dataset.py` no?
If so, why do we have here code for creating img-based dataset?

File: utils/build_video_reid_dataset.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                if set_folder == 'bounding_box_train':  # add to gallery as well
+                    dataset_crop_path = os.path.join(dataset_path, 'bounding_box_test', output_name)
+                    os.makedirs(os.path.join(dataset_path, 'bounding_box_test'), exist_ok=True)
+                    shutil.copy(orig_crop_path, dataset_crop_path)
+                crop_counter += 1
+                df = add_file_data_base(df, output_name, crop)
+    df.to_csv(os.path.join(DATASET_OUTPUT_LOCATION, 'img_to_info.csv'))
+
+
+def convert_to_mars_naming(dataset_path: str):
```
Comment (darkushin) [inline utils/build_video_reid_dataset.py:67]: Can we remove this function or are we planning to use the MARS notation someday?

File: utils/build_video_reid_dataset.py (hunk 1 above)
Comment (bar371) [reply utils/build_video_reid_dataset.py]: that file is no longer relevant for building an image based dataset from our DB

File: utils/build_video_reid_dataset.py (hunk 2 above)
Comment (bar371) [reply utils/build_video_reid_dataset.py:67]: I say keep it, we might want to check on MARS later on

**Review by darkushin — DISMISSED** [review]

File: utils/build_video_reid_dataset.py (hunk 1 above)
Comment (darkushin) [inline utils/build_video_reid_dataset.py]: What is same_day, is it a boolean or a string? you set it here to False as default, but treat it as a string later.
what is it supposed to indicate?

**Review by darkushin — APPROVED** [review]

**Pull request merged**